.gitignore

```diff
@@ -1,3 +1,4 @@
+**/.DS_Store
```

Comment:
Looks like this got swapped under the rug: can we make it say ".DS_Store" without the `**/`?

Reply:
The pattern without the glob should also catch the file in a subdir... Does it not?

Reply:
You are right, probably that 'inner' `.DS_Store` slept into the git before i've added the initial ignore and that made me think that it needs a glob.
Removed.